Taskfile.yaml

```diff
@@ -1,5 +1,8 @@
 version: '3'
 
+env:
+  IMAGE_NAME: '{{ .IMAGE_NAME | default "qniw984/waffle:1.0.0" }}'
```

Comment:
Why qniw984 tho? 😄

Reply:
Changed to `<YOUR_USERNAME>`, so user can specify his/her username